modules/flye/main.nf

```diff
@@ -0,0 +1,57 @@
+process FLYE {
+    tag "$meta.id"
+    label 'process_high'
+
+    conda (params.enable_conda ? "bioconda::flye=2.9" : null)
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/flye:2.9--py39h6935b12_1' :
+        'quay.io/biocontainers/flye:2.9--py39h6935b12_1' }"
+
+    input:
+    tuple val(meta), path(reads)
+    val mode
+
+    output:
+    tuple val(meta), path("*.fasta.gz"), emit: fasta
+    tuple val(meta), path("*.gfa.gz")  , emit: gfa
+    tuple val(meta), path("*.gv.gz")   , emit: gv
+    tuple val(meta), path("*.txt")     , emit: txt
+    tuple val(meta), path("*.log")     , emit: log
+    tuple val(meta), path("*.json")    , emit: json
+    path "versions.yml"                , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+    def prefix = task.ext.prefix ?: "${meta.id}"
```

Comment:
You consider adding a validation check for `mode` as it is critical for the tool to run correctly

https://github.com/nf-core/modules/blob/486eebfd298d6b603106c8def8557b6caf0af58e/modules/malt/build/main.nf#L34-L35